README.md

```diff
@@ -16,63 +16,75 @@ For an introduction to mutation testing and Stryker's features, see [stryker-mut
 
 ## Getting Started
 
-For the quickstart, see [our website's quickstart](https://stryker-mutator.io/stryker4s/quickstart).
+For the quickstart, see [our  quickstart](docs/getting-started.md).
+See our [sample projects](sample-projects) for examples on how to add stryker to your project.
 
-Stryker4s is a mutation testing framework for Scala. It allows you to test your tests by temporarily inserting bugs. Depending on your project setup, there are multiple ways to get started with Stryker4s.
+Stryker4jvm is a mutation testing framework for jvm-languages, currently supporting Scala and Kotlin. It allows you to test your tests by temporarily inserting bugs. 
+
+Depending on your project setup, there are multiple ways to get started with Stryker4jvm.
 
 ## Sbt plugin
 
-Stryker4s provides a sbt plugin for easy use within sbt projects. To install the plugin, add the following line to `plugins.sbt` [![Maven Central](https://img.shields.io/maven-central/v/io.stryker-mutator/stryker4s-core_2.12.svg?label=Maven%20Central&colorB=brightgreen)](https://search.maven.org/artifact/io.stryker-mutator/sbt-stryker4s):
+Stryker4jvm provides a sbt plugin for easy use within sbt projects. To install the plugin, add the following line to `plugins.sbt` [![Maven Central](https://img.shields.io/maven-central/v/io.stryker-mutator/stryker4s-core_2.12.svg?label=Maven%20Central&colorB=brightgreen)](https://search.maven.org/artifact/io.stryker-mutator/sbt-stryker4s):
 
 ```scala
-addSbtPlugin("io.stryker-mutator" % "sbt-stryker4s" % stryker4sVersion)
+addSbtPlugin("io.stryker-mutator" % "stryker4jvm-plugin-sbt" % stryker4jvmVersion)
 ```
 
-After adding the plugin, Stryker4s can be used by running `sbt stryker` in the root of your project.
+After adding the plugin, Stryker4jvm can be used by running `sbt stryker` in the root of your project.
 
 ### Multi-module projects
 
-Multi-module projects are not yet fully supported. However, there is a workaround you can use while we work on a better solution. Set the base-directory to the correct directory of the submodule with the [`base-dir` configuration setting](https://stryker-mutator.io/docs/stryker4s/configuration#base-dir-string). Then you can run `sbt "project yourSubmoduleNameHere" stryker` to set the active project and run Stryker4s.
```

Comment:
In general links should still direct people to the website. Maybe once we've finalized on the name we can also update the website directory (and redirect the old links)